controllers/adminController.js

```diff
@@ -1,11 +1,54 @@
 const helpers = require('../_helpers')
+const db = require('../models')
+const User = db.User
+const Tweet = db.Tweet
+const Like = db.Like
 
 const adminController = {
   readUsers: (req, res) => {
-    res.send(`Hello Admin ${helpers.getUser(req).name}!`)
+    User.findAll({
+      where: { role: 'user' },
+      include: [
+        Tweet,
+        Like,
+        { model: User, as: 'Followers' },
+        { model: User, as: 'Followings' },
+      ]
+    }).then(users => {
+      users = users.map(user => ({
+        ...(Object.fromEntries(Object.entries(user.dataValues).slice(0, 11))),
+        tweetsCount: user.dataValues.Tweets.length,
+        likesCount: user.dataValues.Likes.length,
+        followersCount: user.dataValues.Followers.length,
+        followingsCount: user.dataValues.Followings.length
+      }))
+      users = users.sort((a, b) => b.tweetsCount - a.tweetsCount)
+      return res.json(users)
+    }).catch(err => console.error(err))
+  },
+  readTweets: (req, res) => {
+    Tweet.findAll({
+      include: [{
+        model: User,
+        attributes: ['id', 'account', 'name', 'avatar']
+      }]
+    }).then(tweets => res.json(tweets))
```

Comment:
```suggestion
readTweets: (req, res) => {
    Tweet.findAll({
      include: [{
        model: User,
        attributes: ['id', 'account', 'name', 'avatar']
      }]
    }).then(tweets => res.json(tweets))
```
UserStories上面說
> 可以快覽推文前50字
這邊可能需要用到description.substring(0, 50)來完成。

Reply:
這邊前端跟我要全部文字，他要前端用 [...顯示更多] 來處理

感謝 🙏

Reply:
OK